.expeditor/habitat-test.pipeline.yml

```diff
@@ -1,8 +1,18 @@
 ---
+expeditor:
+  defaults:
+    buildkite:
+      timeout_in_minutes: 60
+      retry:
+        automatic:
+          limit: 1
+
 steps:
 
 - label: ":windows: Validate Habitat Builds of Chef Infra"
   commands:
+    - powershell -File "./scripts/ci/ensure-minimum-viable-hab.ps1"
+    - 'Write-Host "--- :hammer_and_wrench: Installing $$EXPEDITOR_PKG_IDENTS_CHEFINFRACLIENTX86_64WINDOWS"'
```

Comment:
I think `$$EXPEDITOR_PKG_IDENTS_CHEFINFRACLIENTX86_64WINDOWS` isn't getting interpolated properly. One of the builds looked like it was blank.

Reply:
Stupid double dollars! :rage1:

Reply:
Stupid double dollars! :rage1: 

Updated with only one `$`.